iron-meta.d.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * DO NOT EDIT
+ *
+ * This file was automatically generated by
+ *   https://github.com/Polymer/gen-typescript-declarations
+ *
+ * To modify these typings, edit the source file(s):
+ *   iron-meta.html
+ */
+
+/// <reference path="../polymer/types/polymer.d.ts" />
+
+/**
+ * `iron-meta` is a generic element you can use for sharing information across the DOM tree.
+ * It uses [monostate pattern](http://c2.com/cgi/wiki?MonostatePattern) such that any
+ * instance of iron-meta has access to the shared
+ * information. You can use `iron-meta` to share whatever you want (or create an extension
+ * [like x-meta] for enhancements).
+ *
+ * The `iron-meta` instances containing your actual data can be loaded in an import,
+ * or constructed in any way you see fit. The only requirement is that you create them
+ * before you try to access them.
+ *
+ * Examples:
+ *
+ * If I create an instance like this:
+ *
+ *     <iron-meta key="info" value="foo/bar"></iron-meta>
+ *
+ * Note that value="foo/bar" is the metadata I've defined. I could define more
+ * attributes or use child nodes to define additional metadata.
+ *
+ * Now I can access that element (and it's metadata) from any iron-meta instance
+ * via the byKey method, e.g.
+ *
+ *     meta.byKey('info');
+ *
+ * Pure imperative form would be like:
+ *
+ *     document.createElement('iron-meta').byKey('info');
+ *
+ * Or, in a Polymer element, you can include a meta in your template:
+ *
+ *     <iron-meta id="meta"></iron-meta>
+ *     ...
+ *     this.$.meta.byKey('info');
+ */
+interface IronMetaElement extends Polymer.Element {
```

Comment:
Polymer.IronMeta is not being typed, only the element.

https://github.com/PolymerElements/iron-meta/blob/master/iron-meta.html#L66

Reply:
Analyzer doesn't pick up classes defined this way, so added manual typings for it, PTAL!